src/core/positioning/positioning.cpp

```diff
@@ -157,7 +157,7 @@ void Positioning::setupDevice()
     else if ( mDeviceId.startsWith( QStringLiteral( "serial:" ) ) )
     {
       const QString address = mDeviceId.mid( 7 );
-      mReceiver = std::make_unique<SerialPortReceiver>( address );
+      mReceiver = std::make_unique<SerialPortReceiver>( address, this );
```

Comment:
Unique_ptr + qobject with parent is risky.
There should be a QgsObjectUniquePtr (if that's still needed with the parent)

Reply:
@m-kuhn , OK, I think you are right, since we're parenting the QObject here we should just let Qt work its magic.